edx_exams/apps/core/tests/test_api.py

```diff
@@ -623,6 +624,62 @@ def test_exam_with_no_due_date(self):
         self.assertIsNotNone(ExamAttempt.objects.get(user_id=user_id, exam_id=exam_id))
 
 
+class TestDeleteExamAttempt(ExamsAPITestCase):
+    """
+    Tests for the API utility function `delete_exam_attempt`
+    """
+    def setUp(self):
+        super().setUp()
+
+        self.exam = ExamFactory()
+        self.student_user = UserFactory()
+        self.exam_attempt = ExamAttemptFactory(user=self.student_user, exam=self.exam)
+
+    def test_delete_exam_attempt(self):
+        """
+        Test that an exam attempt is deleted
+        """
+        delete_exam_attempt(self.exam_attempt, self.user)
+        self.assertFalse(ExamAttempt.objects.filter(id=self.exam_attempt.id).exists())
+
+    @patch('edx_exams.apps.core.signals.signals.EXAM_ATTEMPT_RESET.send_event')
+    def test_event_emitted(self, mock_event_send):
```

Comment:
Can this be worked into `test_attempt_event_emitted` in this file? Here it is on the [feature branch](https://github.com/edx/edx-exams/pull/191/files#diff-c820205e66b9fd0bec00c57fd07a83600a5e0091f836034b8f90c39135c617d4R309).

Reply:
I think this one is still separate because a reset doesn't go through the useual 'update_attempt_status' flow